CONTRIBUTING.md

```diff
@@ -64,6 +64,12 @@ for it to be reviewed (Status: Dev Complete).
 * Note: if you have write access to the repository, do not directly merge pull 
   requests. Let another team member review your pull request and approve it.
 
+## Before Merge
+* Bump the version into the files; `npm-shrinkwrap.json`, `package.json`, `sonar-project.properties`, e.g. version": "3.2.0-BUILD-NUMBER"
+* Prepend your commit message with a JIRA ticket number and the text Bump Version, e.g. "FH-1234: Bump Version."
```

Comment:
I think the version bump commit doesn't have to be prepended with the JIRA number, only the commites with actual changes.

Reply:
👍 Tks for the review.  Done the change.